.circleci/config.yml

```diff
@@ -227,6 +227,61 @@ workflows:
           private_registry: true
           timeout: '2000'
 
+  nightly_build:
+    triggers:
+      - schedule:
+          cron: "0 10 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
```

Comment:
every job in this workflow should have `filters: *filter_nightly`

Reply:
Will do, thanks!